.github/workflows/pr.yml

```diff
@@ -0,0 +1,14 @@
+name: PR
+on:
+  pull_request:
+    branches:
+      - 'main'
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v4
+        with:
+          go-version: 1.21
+      - run: go test -v ./...
```

Comment:
Can you please add a timeout

Reply:
Looks like the `testcontainers` work on github!
Had problems with them on azure.

Reply:
This time out has now been added, I made it 10 mins just to be generous,